apps/OpenSpace/ext/launcher/src/settingsdialog.cpp

```diff
@@ -0,0 +1,407 @@
+            "Novice User",
+            "User",
+            "Advanced User",
+            "Developer"
+        });
+        _propertyVisibility->setCurrentText("User");
+        connect(
+            _propertyVisibility,
+            &QComboBox::textActivated,
+            [this](const QString& value) {
+                using Visibility = openspace::properties::Property::Visibility;
+                if (value == "Novice User") {
+                    _currentEdit.visibility = Visibility::NoviceUser;
+                }
+                else if (value == "User") {
+                    // This is the default value
+                    _currentEdit.visibility = std::nullopt;
+                }
+                else if (value == "Advanced User") {
+                    _currentEdit.visibility = Visibility::AdvancedUser;
+                }
+                else if (value == "Developer") {
+                    _currentEdit.visibility = Visibility::Developer;
+                }
+                else {
+                    throw ghoul::MissingCaseException();
+                }
+
+                updateSaveButton();
+            }
+        );
+        layout->addWidget(_propertyVisibility, 9, 1);
+
+        _bypassLauncher = new QCheckBox("Bypass Launcher");
+        _bypassLauncher->setToolTip(
+            "If this value is selected, the Launcher will no longer be shown at startup. "
+            "Note that this also means that it will not be easy to get back to this "
+            "setting to reenable the Launcher either."
+        );
+        connect(
+            _bypassLauncher,
+            &QCheckBox::stateChanged,
+            [this]() {
+                if (_bypassLauncher->isChecked()) {
+                    _currentEdit.bypassLauncher = _bypassLauncher->isChecked();
+                }
+                else {
+                    _currentEdit.bypassLauncher = std::nullopt;
+                }
+                _bypassInformation->setVisible(_bypassLauncher->isChecked());
+                updateSaveButton();
+            }
+        );
+        layout->addWidget(_bypassLauncher, 10, 0, 1, 2);
+
+        _bypassInformation = new QLabel(
+            "Saving the settings with the bypass launcher enabled will cause this window "
+            "to not show up again, making it harder to undo this change. In case you "
+            "need to undo it, you need to open the settings.json and remove the line "
+            "that says '\"bypass\": true,'"
+        );
+        _bypassInformation->setObjectName("information");
+        _bypassInformation->setHidden(true);
+        _bypassInformation->setWordWrap(true);
+        layout->addWidget(_bypassInformation, 11, 0, 1, 2);
+    }
+
+    layout->addWidget(new Line(), 12, 0, 1, 2);
+
+    {
+        QLabel* label = new QLabel("MRF Caching");
+        label->setObjectName("heading");
+        layout->addWidget(label, 13, 0, 1, 2);
+
+        _mrf.isEnabled = new QCheckBox("Enable caching");
```

Comment:
```suggestion
        _mrf.isEnabled = new QCheckBox("Enable Caching");
```
to be consistent with the other labels